### PR TITLE
Program to count digits in an integer

### PR DESCRIPTION
**Pull request by Sapna2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes.

### Description
Program to count digits in an integer 

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#406

### CODE REVIEW

File: Python/README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@
 * [Sieve of Eratosthenes](math/prime_number_using_sieve_of_Eratosthenes.py)
 * [Fibonacci number](math/Fibonacci.py)
 * [Strassen's Matrix Multiplication](math/Strassen's_Matrix_Mul.py)
+* [No:of digits in a number](math/no_of_digits.py)
```
Comment (atarax665) [inline Python/README.md]: Change it to '[No. of Digits in an Integer]' .

File: Python/math/no_of_digits.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    return 1 + recursive_countDigits(n // 10) 
+
+if __name__ == "__main__":
+    n = int(input("Enter the number: "))
+    print("Iterative approach")
+    print("The no: of digits in {} is: {} ".format(n, iterative_countDigits(n)))
+    print("Recursive approach")
+    print("The no: of digits in {} is: {} ".format(n, recursive_countDigits(n)))
+
+
```
Comment (atarax665) [inline Python/math/no_of_digits.py]: Remove this extra line. Everything else looks fine to me.

File: Python/math/no_of_digits_in_an_integer.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+#Recursive approach
+def recursive_countDigits(n): 
+    if n == 0: 
+        return 0
+    return 1 + recursive_countDigits(n // 10) 
+
+if __name__ == "__main__":
+    n = int(input("Enter the number: "))
+    print("Iterative approach")
+    print("The no: of digits in {} is: {} ".format(n, iterative_countDigits(n)))
```
Comment (Kajol-Kumari) [inline Python/math/no_of_digits_in_an_integer.py:20]: Please change `no:` to `no.`

File: Python/math/no_of_digits_in_an_integer.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    print("Recursive approach")
+    print("The no: of digits in {} is: {} ".format(n, recursive_countDigits(n)))
```
Comment (Kajol-Kumari) [inline Python/math/no_of_digits_in_an_integer.py:22]: Same here `no:` to `no.`

**Review by Kajol-Kumari — APPROVED** [review]
@Sapna2001 please incorporate the minor changes requested, rest looks good to me :)

**Pull request closed without merging**